### ISSUE TITLE
Import HD map Lanelet2 fail? InvalidOperationExecption: Nullable object must have a value. 

### ISSUE THREAD

**Issue by llu0120** [lgsvl/simulator#1987]
I'm trying to use the lgsvl simulator and I'm using the `tag 2021.3` in the `simulator` repo. However, when I'm trying to import my lanelet2 map into the simulator, I can't see my map inside unity and report an error below. Has anyone confront this before and can help with this? Thanks in advance. 
 
![image](https://user-images.githubusercontent.com/41609738/154344568-1d1b6690-bcd2-4f4f-9f6c-f7dab3aad500.png)

**Comment by EricBoiseLGSVL** [comment]
You most likely have an error in the hdmap or an corner case that the importer can't handle.  You will need to trace the import and see what part is causing the exception.

**Comment by llu0120** [comment]
Hi @EricBoiseLGSVL, 
Thanks for your reply. I've tried to import the example lanelet2 map from BorregasAve, but the import visualization looks strange as well. 
![image](https://user-images.githubusercontent.com/41609738/155014276-0c1792be-a0c4-487c-a9a5-c3ebfdc23b34.png)

Looks like only the traffic lights are being imported. Is it normal? Or how can I visualize the lanes, boundaries...etc? 

Thanks in advance

**Comment by EricBoiseLGSVL** [comment]
Do you have generate mesh setting enabled in the import settings?

**Comment by llu0120** [comment]
no, but I can see the lanes when clicking view all in the HD Map Annotation mode. Is that normal? Thanks!

**Comment by EricBoiseLGSVL** [comment]
Yes, this is just annotation data, not a drive-able mesh.
